contents/blog/5-ways-to-improve-analytics-data.md

```diff
@@ -0,0 +1,132 @@
+  - remove
+  - start
+  - end
+  - cancel
+  - fail
+  - generate
+  - send
+  ```
+
+- For event names, use the `category:object_action` framework:
+
+  - `category` describes the context the event took place in – e.g., `account_settings` or `signup_flow` 
+
+  - `object` is a noun that refers to the component or location the event took place in, like `forget_password_button` or `pricing_page`.
+
+  - `action` is a verb that describes what happened, like `click`, `submitted`, or `create`
+
+  - Putting all of these together, you get `account_settings:forget_password_button_click` or `signup_flow:pricing_page_view` – The order is important as it makes it easier to find a specific event within a long list.
+
+- For property names:
+
+  - Use the `object_adjective` pattern. For instance: `user_id`, `item_price`, or `member_count`.
+
+  - Use `is/has` prefixes for boolean properties. For example: `is_subscribed`, `has_seen_upsell`.
+
+  - If the property's value is a date or timestamp, include _date or _timestamp at the end. For example, `user_creation_date`, `last_login_timestamp`.
+
+## 2. Filter out internal users
+
+The most common mistake we see is developers inadvertently inflating their own metrics. When building or testing their apps, they send many events but forget to filter them out afterward. This leads to massive bias in their data!
+
+For this reason, it's important to filter out events sent by your own team. There are a few ways to do this:
+
+- Filter out internal emails.
+- Add a property on your events `is_employee` or `is_test_user` and filter events where these are `true`.
+- Exclude internal IPs from your analytics tracking.
+- Filter events by domain host e.g., exclude `localhost:3000`, `staging.yourapp.com`.
+- Turn off tracking in development using a `localhost`, `dev`, or config check.
+
+> **Tip:** Rather than add these filters to each new [insight](/docs/product-analytics/insights) you create, set it as your default filter.
+> 
+> For a detailed guide on how to do this in PostHog, see the tutorial on [how to filter out internal users in PostHog](/tutorials/filter-internal-users).
+
+## 3. Prefer backend to frontend tracking
+
+Backend analytics are more reliable than frontend analytics. There are 3 reasons for this:
+
+1. Many users have tracking disabled or blocked on their browsers (especially tech-savvy users).
+2. Frontend analytics often rely on JavaScript execution, which can be interrupted by various factors – such as network issues, CORS, browser settings, and more.
+3. You have complete control of your backend implementation and execution.
+
+Thus, when possible, log events on your server instead of on your client.
+
+That doesn't mean that frontend analytics are entirely useless, though. As long as you know they're only showing a portion of your data, they can still be useful. 
+
+Below is a guide to help you decide when to rely on frontend or backend analytics:
+
+- Use frontend analytics if you want to:
```

Comment:
Softened this. I didn't want it to sound like frontend events are inferior, when a lot of your product is based on it!